config/config.py

```diff
@@ -26,25 +26,25 @@
 
 def load_config(config_file=CONFIG_FILE, error=True):
     logger = logging.getLogger("CONFIG LOADER")
-    basic_error = "Error when load config file {0}".format(config_file)
+    basic_error = f"Error when load config file {config_file}"
```

Comment:
I don't think f-string migration on this file is a good idea. It's the first module been imported by start.py at startup, before `tools/__init__.py` that check python compatibility.
So i suggest to revert this modifications **OR** import `tools/__init__.py` before.